hkp_test.go

```diff
@@ -67,7 +67,7 @@ func Test_index(t *testing.T) {
 		t.Fatalf("Client.Index(): %v", err)
 	}
 
-	creationTime, _ := time.Parse(time.RFC3339, "2013-07-20T18:32:38+02:00")
+	creationTime, _ := time.Parse(time.RFC3339, "2013-07-20T12:32:38-04:00")
```

Comment:
Can you try keeping the original date string, but adding `.Local()` to convert it to the local timezone?